api/swig/python/setup.py.in

```diff
@@ -0,0 +1,35 @@
+import setuptools.command.install
+import shutil
+from distutils.sysconfig import get_python_lib
+
+
+class CompiledLibInstall(setuptools.command.install.install):
+    """
+    Specialized install to install to python libs
+    """
+
+    def run(self):
+        """
+        Run method called by setup
+        :return:
+        """
+        # Get filenames from CMake variable
+        filenames = '${PYTHON_INSTALL_FILES}'.split(';')
+
+        # Directory to install to
+        install_dir = get_python_lib()
```

Comment:
Given that there is also a _bagPy.pyd and baglib.dll to install, wouldn't it be better to have all under a bagPy package folder? The current way puts all the 3 files under site-packages.

Reply:
Agreed.

Reply:
Will address in version 2.0.2, see: https://github.com/OpenNavigationSurface/BAG/issues/69.